keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,7 +385,28 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+
+#### How to Utilize the API
+This section provides guidance on the recommended way to interact with the API. It is suggested that
+all clusterProfile objects, which represent a single cluster in the inventory, should be emitted
+on a dedicated management/hub Kubernetes cluster. This approach allows consumers to have a single
+integration point to access all the information within a cluster inventory. A controller can be run 
+on the management or hub cluster to offer high-level functionalities over this inventory of clusters.
+
+#### Approaches for Structuring Clusters within an Inventory
+This section provides recommendations on the optimal way to structure clusters within an inventory. 
+It's important to note that while there are no strict requirements, we make the clusterProfile API a 
+namespace-scoped object. This approach allows users to leverage Kubernetes' native namespace-based
+RBAC if they wish to restrict access to certain sub-section of clusters with in the inventory.
+
+#### Strategies for Organizing Multiple Cluster Inventories
+This section offers guidance on the optimal methods for organizing multiple cluster inventories. While there are no
+strict requirements, if users opt to use a single management/hub Kubernetes cluster to manage multiple cluster inventories, 
+it is strongly recommended to place all the clusterProfile objects belonging to the same cluster inventory in the same namespace.
+This approach enforces the principle of namespace sameness within each cluster inventory. However, it's important to note
+that this means users would lose the ability to apply Kubernetes' native namespace-based RBAC within an inventory since
+Kubernetes does not support nested namespaces.
```

Comment:
```suggestion
This section aims to guide the best practices for organizing multiple cluster inventories. While there are no strict requirements, it is recommended that users use a single management/hub Kubernetes cluster to manage multiple cluster inventories. If this approach is used, it is strongly advised to keep all the `ClusterProfile` objects of the same cluster inventory in the same namespace. This practice ensures namespace sameness within each cluster inventory. However, it is important to note that this approach will result in the inability to apply Kubernetes' native namespace-based RBAC within an inventory since Kubernetes does not support nested namespaces.
```